source/common/router/config_impl.cc

```diff
@@ -431,6 +433,15 @@ RouteEntryImplBase::RouteEntryImplBase(const VirtualHostImpl& vhost,
     regex_rewrite_ = Regex::Utility::parseRegex(rewrite_spec.pattern());
     regex_rewrite_substitution_ = rewrite_spec.substitution();
   }
+
+  if (Runtime::runtimeFeatureEnabled(
```

Comment:
@mattklein123  Finally I found out the reason why the test `config_test:example_configs_test` panicked is that I invoke `Runtime::runtimeFeatureEnabled` here, looks like we cannot call this method in the constructor.

Reply:
I think it's missing "mocking" the `threadsafeSnapshot()` for `server_.runtime_loader_`.

https://github.com/envoyproxy/envoy/pull/11667/files#diff-cee698f5ff6e31cd2ec9d1f978efa79d

The good news is you can merge master and get that fix (since it is part of https://github.com/envoyproxy/envoy/pull/11667).

Reply:
Oh great! Thank you!